Wikipedia/Code/DescriptionEditViewController.swift

```diff
@@ -181,10 +181,12 @@ class DescriptionEditViewController: WMFScrollViewController, Themeable, UITextV
         enableProgressiveButton(false)
         wmf_hideKeyboard()
         
-        // Final trim to remove leading and trailing space
-        let descriptionToSave = descriptionTextView.text.trimmingCharacters(in: .whitespacesAndNewlines)
-        
-        guard let article = article, let dataStore = article.dataStore, let articleURL = article.url else {
+        guard
+            let descriptionToSave = descriptionTextView.normalizedWhitespaceText(),
```

Comment:
If this is nil, do we want to show an error to the user?

Reply:
checking...

Reply:
(nil or an empty string)

Reply:
fixing...